src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -25,6 +27,16 @@
  */
 public class UpdateFeedbackResponseActionTest extends BaseActionTest<UpdateFeedbackResponseAction> {
 
+    private StudentAttributes student1InCourse1;
+    private StudentAttributes student2InCourse1;
+    private StudentAttributes student5InCourse1;
+    private InstructorAttributes instructor2OfCourse1;
```

Comment:
This is only utilized in `testAccessControl_instructorAccessOtherGiver_inaccessible`. It's also used in unknown intent, but any other instructor / student will do too.